Lib/test/test_builtin.py

```diff
@@ -1475,11 +1614,16 @@ def test_bug_27936(self):
             self.assertEqual(round(x, None), round(x))
             self.assertEqual(type(round(x, None)), type(round(x)))
 
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
```

Comment:
fixing this one is looking like a good first issue

Reply:
@DimitrisJim I believe this should have been broken out into a separate commit.